.github/workflows/e2e.yml

```
@@ -0,0 +1,84 @@
+name: e2e suite
+
+on:
+  workflow_dispatch:
+  schedule:
+    # At 2:00 AM UTC on Monday, Wednesday, and Friday
+    - cron: "00 2 * * 1,3,5"
+  pull_request:
+    branches:
+      - 'main'
+
+env:
+  BENCH_PATH: ./queries-results.json
```

`BENCH_PATH` isn't used:

```suggestion
env:
```